app/src/main/java/ch/epfl/qedit/model/answer/EmptyAnswerModel.java

```diff
@@ -1,4 +1,11 @@
 package ch.epfl.qedit.model.answer;
 
+import androidx.annotation.Nullable;
+
 /** This model contains nothing */
-public class EmptyAnswerModel extends AnswerModel {}
```

Comment:
With my PR merged you won't need to implement the method as I removed EmptyAnswerFormat and everything that goes with it.

Reply:
Okok I will delete this too, thanks